keylime/tpm/tpm_main.py

```diff
@@ -477,7 +478,7 @@ def __create_aik(
                     cmd = ["tpm2_evictcontrol", "-A", "o", "-H", hex(cast(int, aik_handle)), "-P", owner_pw]
                     retDict = self.__run(cmd, raiseOnError=False)
                 else:
-                    cmd = ["tpm2_evictcontrol", "-C", "o", "-c", aik_handle, "-P", owner_pw]
+                    cmd = ["tpm2_evictcontrol", "-C", "o", "-c", cast(str, aik_handle), "-P", owner_pw]
```

Comment:
maybe do in line 449 a `aik_handle = str(self.get_tpm_metadata("aik_handle"))`

Reply:
aik_handle can also be an int on older versions of the tools: `cmd = ["tpm2_evictcontrol", "-A", "o", "-H", hex(cast(int, aik_handle)), "-P", owner_pw]`